src/node/Params.hs

```diff
@@ -123,26 +129,23 @@ getNodeParams args@Args {..} systemStart = do
         , npEnableMetrics = enableMetrics
         , npEkgParams = ekgParams
         , npStatsdParams = statsdParams
+        , npRelayParams = error "TODO define the RelayParams"
```

Comment:
@edsko This had me confused. The commit which introduces `RelayParams` and adds this field `npRelayParams` doesn't define the field here, so I put in the `error` to make it compile.